### PR TITLE
Abort task

### PR DESCRIPTION
**Pull request by barwepaul**
- Get tasks from control-server
- Abort task
- Update task status on page refresh

### CODE REVIEW

File: web-server/django_web_server/website/server.py [hunk 1]
```diff
@@ -41,22 +46,59 @@ def getClients(self):
...
+    def __saveTask(self, t_id, c_id, task_t, task_i, t_status):
+        if task_t != 'abort':
+            client = Client.objects.get(client_id=c_id)
+            t = time.localtime()
+            asc_t = time.asctime(t)
+            client.senttask_set.create(task_id=t_id, start_time=asc_t, status=t_status,
+                                       task_type=task_t, task_info=task_i)
+
+    def getTasks(self):
+        # TODO: Update when endpoint alltasks is implemented
```
Comment (ghmo2789) [inline web-server/django_web_server/website/server.py:58]: This endpoint is already implemented.

**Review by ghmo2789 — CHANGES_REQUESTED** [review]
For most part everything looks good. I needed to install django-filters and django-request before being able to run server. Maybe add those two package in requirements.txt file.

File: web-server/django_web_server/website/server.py (hunk 1 above)
Comment (izaalstergren) [reply web-server/django_web_server/website/server.py:58]: Nice! We updated the code to only make one request to load all tasks

**Review by ghmo2789 — APPROVED** [review]

File: web-server/django_web_server/website/server.py (hunk 1 above)
Comment (johanlovgren) [reply web-server/django_web_server/website/server.py:58]: Is the code updated? Then it's maybe possible to remove the TODO

**Review by johanlovgren — APPROVED** [review]
Looks good! 
Just a minor comment/question.

**Pull request merged**